crates/rollup/src/cli.rs

```diff
@@ -8,6 +8,9 @@ use url::Url;
 /// The default L2 chain ID to use. This corresponds to OP Mainnet.
 pub const DEFAULT_L2_CHAIN_ID: u64 = 10;
 
+/// The default L1 RPC URL to use.
+pub const DEFAULT_L1_RPC_URL: &str = "https://eth.llamarpc.com/";
```

Comment:
It would be really neat if we could verify that the given RPC supports the method endpoints needed by `kona-derive`.

Reply:
Do you know which methods kona uses outside of the eth_ namespace? 
I'll try reach out to llamanodes if so, otherwise we can put `http://localhost:8545` to be safe

Reply:
Will write up a ticket for this 👍